packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -740,11 +745,16 @@ export class ListItem
     this.handleOpenableChange(event.target as HTMLSlotElement);
   };
 
-  toggleOpen = (): void => {
-    this.open = !this.open;
+  handleToggleOpenClick = (): void => {
+    this.toggleOpen();
+  };
+
+  toggleOpen = (value = !this.open): void => {
```

Comment:
For readability, WDYT about renaming this to `toggle`?

#### before

`toggleOpen()`
`toggleOpen(true)`
`toggleOpen(false)`

#### after

`toggle()`
`toggle(true)`
`toggle(false)`